templates/commonsense_qa/templates.yaml

```diff
@@ -0,0 +1,111 @@
+dataset: commonsense_qa
+templates:
+  1e1d0ce1-b0ea-4ad8-9971-b2b44948123b: !Template
+    id: 1e1d0ce1-b0ea-4ad8-9971-b2b44948123b
+    jinja: '
+
+      {{question}} |||
+
+      {{ choices[''text''][choices[''label''].index(answerKey)] }}'
+    name: not_multiple_choice
+    reference: ''
+  41188da5-c16a-4c6b-89af-6ce6815aedc6: !Template
+    id: 41188da5-c16a-4c6b-89af-6ce6815aedc6
+    jinja: '
+
+      {{question}}
+
+
+      {% for i in range(choices[''text'']|length) %}
+
+      - {{choices[''text''][i]}}
+
+      {% endfor %} |||
+
+      {{ choices[''text''][choices[''label''].index(answerKey)] }}'
+    name: qa
+    reference: ''
+  42fca80b-b614-4288-aad2-2525360543cb: !Template
+    id: 42fca80b-b614-4288-aad2-2525360543cb
+    jinja: 'Pick the most suitable answer for the following question:
+
+
+      {{question}}
+
+
+      Options:
+
+
+      {% for i in range(choices[''text'']|length) %}
+
+      <b>{{choices[''label''][i]}}</b>: {{choices[''text''][i]}}
```

Comment:
Can you remove this HTML markup? The model will only be able to process plain text.